packages/browser/src/Component/InputManager.js

```diff
@@ -16,8 +16,10 @@ const Command = require('@ud-viz/shared').Command;
 export class InputManager {
   /**
    * Manage user inputs with a poll system (https://en.wikipedia.org/wiki/Polling_(computer_science))
+   *
+   * @param {boolean} [isCaseSensitive=false] true so that the inputs are case sensitive
    */
-  constructor() {
+  constructor(isCaseSensitive = true) {
```

Comment:
jsdoc default param is false and the default value in constructor is true. According what you said looks like the default behavior should be false.

Reply:
Good point